packages/material-ui-styles/src/index.d.ts

```diff
@@ -13,7 +13,12 @@ declare module '@material-ui/styles' {
   export { default as StylesProvider } from '@material-ui/styles/StylesProvider';
   export { default as ThemeProvider } from '@material-ui/styles/ThemeProvider';
   export { default as useTheme } from '@material-ui/styles/useTheme';
-  export { default as withStyles, StyleRules, WithStyles } from '@material-ui/styles/withStyles';
```

Comment:
Huh I've been telling folks to not use `StyleRules` while we're exporting them. I guess it does make sense to export the return types so that one can use explicit function signatures